integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/BridgeSteps.java

```diff
@@ -29,73 +31,83 @@ public BridgeSteps(TestContext context) {
         this.context = context;
     }
 
-    @Given("get list of Bridge instances fails with HTTP response code (\\d+)$")
-    public void authenticationIsEnabled(int responseCode) {
+    @Given("^the list of Bridge instances fails with HTTP response code (\\d+)$")
+    public void listOfBridgeInstancesReturnsHTTPResponseCode(int responseCode) {
         BridgeResource.getBridgeListResponse(context.getManagerToken())
                 .then()
                 .statusCode(responseCode);
     }
 
-    @When("create a Bridge with randomly generated name with access token")
-    public void createRandomBridge() {
-        String bridgeName = Utils.generateId("bridge");
+    @When("^a new Bridge \"([^\"]*)\" is created$")
+    public void newBridgeIsCreated(String testBridgeName) {
+        String systemBridgeName = Utils.generateId("bridge");
         int creationRetry = 1;
-        while (creationRetry <= BRIDGE_NAME_CREATE_RETRY && isBridgeExisting(bridgeName)) {
+        while (creationRetry <= BRIDGE_NAME_CREATE_RETRY && isBridgeExisting(systemBridgeName)) {
             creationRetry++;
-            bridgeName = Utils.generateId("bridge");
+            systemBridgeName = Utils.generateId("bridge");
         }
-        if (isBridgeExisting(bridgeName)) {
+        if (isBridgeExisting(systemBridgeName)) {
             throw new RuntimeException("Cannot create a initiate a random bridge name correctly. Please cleanup the environment...");
         }
 
-        BridgeResponse response = BridgeResource.addBridge(context.getManagerToken(),
-                bridgeName);
-        assertThat(response.getName()).isEqualTo(bridgeName);
+        BridgeResponse response = BridgeResource.addBridge(context.getManagerToken(), systemBridgeName);
+        assertThat(response.getName()).isEqualTo(systemBridgeName);
         assertThat(response.getStatus()).isEqualTo(BridgeStatus.ACCEPTED);
         assertThat(response.getEndpoint()).isNull();
         assertThat(response.getPublishedAt()).isNull();
         assertThat(response.getHref()).isNotNull();
         assertThat(response.getSubmittedAt()).isNotNull();
 
-        context.setRandomBridgeName(bridgeName);
-        context.setBridgeId(response.getId());
+        BridgeContext bridgeContext = context.newBridge(testBridgeName, systemBridgeName);
+        bridgeContext.setBridgeId(response.getId());
+        bridgeContext.setBridgeName(response.getName());
     }
 
-    @Then("get list of Bridge instances with access token contains Bridge with randomly generated name")
-    public void testRandomBridgeExists() {
+    @And("^the list of Bridge instances contains the Bridge \"([^\"]*)\"+$")
```

Comment:
The reason I introduced the access token mention is to provide information for readers of scenario. Without that information nobody could know why this step pass, while step `the list of Bridge instances fails with HTTP response code` fails.

Reply:
I keep forgetting about the `authentication is done against Manager` step, this comment can be ignored.